### PR TITLE
Update tests after automated retries

### PR DESCRIPTION
**Pull request by Lipt0nas**
Notable changes:
scenario4-7,9,13 Deleted (sender side rejections)
scenario10-1,2 Disabled for now, since they should be relevant again after implementing events for transfer resumes/stops
scenario10-3 Deleted as it's no longer relevant

### CODE REVIEW

File: test/scenarios.py [hunk 1]
```diff
@@ -247,18 +247,6 @@
                                 "extension": "none",
                                 "size_kb": 10240
                             }
-                        }""",
```
Comment (matszczygiel) [inline test/scenarios.py:250]: Nice :+1:

File: test/scenarios.py [hunk 2]
```diff
@@ -1430,30 +1275,6 @@
             ),
         },
     ),
-    Scenario(
```
Comment (matszczygiel) [inline test/scenarios.py:1433]: It's not entirely irrelevant. We might actually want to test this behavior

File: test/scenarios.py [hunk 3]
```diff
@@ -2169,152 +2002,132 @@
...
-                            0,
-                            Error.CANCELED,
-                        )
-                    ),
-                    action.NoEvent(),
-                ]
-            ),
-        },
-    ),
-    Scenario(
```
Comment (matszczygiel) [inline test/scenarios.py:2297]: In this one we just expect `Queued` and no further events

File: test/scenarios.py [hunk 4]
```diff
@@ -5360,10 +5163,10 @@
                     action.Wait(event.Start(0, FILES["testfile-big"].id)),
                     action.RejectTransferFile(0, FILES["testfile-big"].id),
                     action.Wait(
-                        event.FinishFileCanceled(0, FILES["testfile-big"].id, False)
+                        event.FinishFileRejected(0, FILES["testfile-big"].id, False)
                     ),
                     action.Wait(
-                        event.FinishFileRejected(0, FILES["testfile-big"].id, False)
+                        event.FinishFileCanceled(0, FILES["testfile-big"].id, False)
                     ),
```
Comment (matszczygiel) [inline test/scenarios.py:5170]: Let's use wait racy, because the order of this events is more or less undefined

File: test/scenarios.py [hunk 5]
```diff
@@ -5452,10 +5255,10 @@
                     action.Wait(event.Start(0, FILES["testfile-big"].id)),
                     action.RejectTransferFile(0, FILES["testfile-big"].id),
                     action.Wait(
-                        event.FinishFileCanceled(0, FILES["testfile-big"].id, False)
+                        event.FinishFileRejected(0, FILES["testfile-big"].id, False)
                     ),
                     action.Wait(
-                        event.FinishFileRejected(0, FILES["testfile-big"].id, False)
+                        event.FinishFileCanceled(0, FILES["testfile-big"].id, False)
```
Comment (matszczygiel) [inline test/scenarios.py:5261]: Same with wait racy

File: test/scenarios.py [hunk 6]
```diff
@@ -5782,22 +5585,9 @@
...
-                            {
-                                event.File(
-                                    FILES["testfile-big"].id,
-                                    "testfile-big",
-                                    10485760,
-                                ),
-                            },
-                        )
-                    ),
                     action.Wait(event.Start(0, FILES["testfile-big"].id)),
```
Comment (matszczygiel) [inline test/scenarios.py:5591]: I'm not sure if this is correct. We expect `Start` to be emitted only once I guess.
We don't have `Paused` event to matchup with `Start`.

File: test/scenarios.py [hunk 7]
```diff
@@ -5834,13 +5624,6 @@
...
                     ),
                     action.Wait(event.Start(0, FILES["testfile-big"].id)),
-                    action.Wait(
-                        event.FinishFailedTransfer(
-                            0,
-                            Error.WS_SERVER,
-                            ignore_os=True,
-                        )
-                    ),
                     action.Wait(event.Start(0, FILES["testfile-big"].id)),
```
Comment (matszczygiel) [inline test/scenarios.py:5627]: Same here I would say that the second `Start` is not desired

File: test/scenarios.py [hunk 8]
```diff
@@ -5885,21 +5668,6 @@
...
-                            {
-                                event.File(
-                                    FILES["testfile-big"].id,
-                                    "testfile-big",
-                                    10485760,
-                                ),
-                            },
-                        )
-                    ),
                     action.Wait(event.Start(0, FILES["testfile-big"].id)),
```
Comment (matszczygiel) [inline test/scenarios.py:5671]: Same with second `Start`

**Comment by matszczygiel** [comment]
Great job. Am I right that the commented-out tests will be handled in the second run?

File: test/scenarios.py (hunk 6 above)
Comment (Lipt0nas) [reply test/scenarios.py:5651]: Then i guess this is a bug in the implementation if we're shooting out start events every time

Comment (matszczygiel) [reply test/scenarios.py:5651]: Yeah, it might be

**Comment by Lipt0nas** [comment]
The two disabled tests expect to get an error once the other sides goes dark, the idea was to repurpose them when we implement the resume/pause events, right now i don't think they would serve a purpose

**Review by matszczygiel — APPROVED** [review]
Regardless of Started events, I think it can be safely merged

**Pull request merged**